ksqldb-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -701,11 +702,11 @@ public void shouldPrintConnectorPluginsList() {
           + "  \"warnings\" : [ ]," + NEWLINE
           + "  \"connectorsPlugins\" : [ {" + NEWLINE
           + "    \"className\" : \"clazz1\"," + NEWLINE
-          + "    \"type\" : \"source\"," + NEWLINE
+          + "    \"type\" : \"SOURCE\"," + NEWLINE
```

Comment:
How come this is uppercased now? It looks like the new PluginType class that got copied over also has `toLowerCase()` in its `toString()` method.

Reply:
`PluginType` was missing the `JsonValue` annotation so it wasn't even calling `toString` 🤦 fixed